CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ pkg_check_modules(GLIB REQUIRED gio-unix-2.0>=2.46.0)
 include_directories(${GLIB_INCLUDE_DIRS})
 pkg_check_modules(JSONC REQUIRED json-c)
 include_directories(${JSONC_INCLUDE_DIRS})
-pkg_check_modules(LIBMODULEMD REQUIRED modulemd)
+pkg_check_modules(LIBMODULEMD REQUIRED modulemd-2.0)
```

Comment:
This definitely needs to have the minimum version set like we have in the spec...